quarkchain/cluster/cluster_config.py

```diff
@@ -92,6 +92,7 @@ class SlaveConfig(BaseConfig):
     WEBSOCKET_JSON_RPC_PORT = None
     ID = ""
     CHAIN_MASK_LIST = None
+    TYPE = ""
```

Comment:
can set the default type here instead of the constructor

Reply:
done